Formula/vim.rb

```diff
@@ -82,13 +81,18 @@ def install
       opts << "--without-x"
     end
 
-    if build.with? "lua"
-      opts << "--enable-luainterp"
-    end
-
-    if build.with? "luajit"
-      opts << "--with-luajit"
+    if build.with?("lua") || build.with?("luajit")
+      ENV["LUA_PREFIX"] = HOMEBREW_PREFIX
       opts << "--enable-luainterp"
+      opts << "--with-luajit" if build.with? "luajit"
```

Comment:
Hilariously, no. If you don't pass `--enable-luainterp` first it'll ignore that you ever mentioned `luajit`.